adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -124,29 +124,27 @@ class Transform(protected val args: TransformArgs) extends BDGSparkCommand[Trans
 
   val stringency = ValidationStringency.valueOf(args.stringency)
 
-  def apply(rdd: RDD[AlignmentRecord],
-            sd: SequenceDictionary,
-            rgd: RecordGroupDictionary): RDD[AlignmentRecord] = {
+  def apply(rdd: AlignmentRecordRDD): AlignmentRecordRDD = {
```

Comment:
`rdd` &rarr; `alignmentRecords` here, to reduce confusion about `rdd.rdd.context` line below